docs/topics_5/installing-web-console-on-openshift.adoc

```diff
@@ -81,6 +92,12 @@ Two templates are provided, one for shared storage and one without shared storag
 . Switch to the *Cluster Console*.
 . Click *Workloads* -> *Pods* and verify that the {ProductShortName} pods are running.
 . Click *Networking* -> *Routes* and then click the URL beside the `mta-web-console` application to open the {WebName} in a new browser window.
++
+[NOTE]
+====
+The instructions above point your browser to an HTTP address that does not support viewing static reports.
```

Comment:
http or https allow to view exactly the same.
choosing one or the other is only a matter of user choice, with the same results , but different security